entrypoint.sh

```diff
@@ -22,7 +22,17 @@ SERVER=""
 [ -n "${STAGING:-}" ] && SERVER="--server https://acme-staging.api.letsencrypt.org/directory"
 
 # Generate strong DH parameters for nginx, if they don't already exist.
-[ -f /etc/ssl/dhparams.pem ] || openssl dhparam -out /etc/ssl/dhparams.pem 2048
+if [ ! -f /etc/ssl/dhparams.pem ]; then
+  if [ -f /cache/dhparams.pem ]; then
+    cp /cache/dhparams.pem /etc/ssl/dhparams.pem
+  else
+    openssl dhparam -out /etc/ssl/dhparams.pem 2048
+    # Cache to a volume for next time?
+    if [ -d /cache ]; then
+      cp /etc/ssl/dhparams.pem /cache/dhparams.pem
+    fi
+  fi
+fi
```

Comment:
Agreed x 2. Like you said, a user has to go out of their way to enable this. At the same time it lets me do continuous deployments when I need to.